environment.yml

```diff
@@ -32,15 +31,16 @@ dependencies:
     - twine
     - jupyter
     - mlflow
-    - torch==1.11.0
-    - torchvision==0.12.0
-    - horovod==0.25.0
+    - numpy
```

Comment:
nit: peg numpy version

Reply:
pinning the numpy to a version is causing the compatability issues.. Spark itself is managing to download the appropriate versions based on the requirements of other dependency packages